docs/release-source/release/stubs.md

```diff
@@ -213,7 +221,11 @@ stub.resetHistory();
 stub.called // false
 ```
 
-*Since `sinon@2.0.0`*
+
+*Since `sinon@5.0.0`*
+
+You can reset behaviour of all stubs using `sinon.resetHistory()`
```

Comment:
Typo: `resetHistory` doesn't reset behavior.

Reply:
> Also do we consistently use the Old English (British?) spelling of behavior in the docs? The actual method name uses the American English spelling.

My natural language uses (mostly) British English spelling. Sometimes when writing code and documentation, that spelling slips through.

(I wonder if there's a spelling linter where you can set the language preference)